client/src/styles/index.css

```diff
@@ -1,13 +1,81 @@
+:root {
```

Comment:
Where does this file get used? It doesn't seem to get imported anywhere?

Reply:
in index.tsx - it just styles the whole page